charts/project-origin-wallet/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for project-origin-wallet.
+
+# image defines the image configuration for the wallet server
+image:
+
+  # repository defines the image repository to use
+  repository: ghcr.io/project-origin/wallet-server
+
+  # tag defines the image tag to use, defaults to the chart appVersion
+  tag:
```

Comment:
Is it possible to set a default value for the tag?

Reply:
it is set through the AppVersion in the chart, this is only to document that override is allowed.